src/js/components/Table/NavigableTable.jsx

```diff
@@ -48,16 +48,19 @@ function NavigableTable({
   const arrowLeftRef = useRef(null)
   const arrowRightRef = useRef(null)
 
-  if (searchParams.get('v') && !slideOverOpen && data.length > 0) {
-    setSlideOverOpen(true)
-    setShowArrows(true)
-    setSelectedIndex(
-      data.findIndex((row) => extractSearchParam(row) === searchParams.get('v'))
-    )
-  } else if (!searchParams.get('v') && slideOverOpen) {
+  if (!searchParams.get('v') && slideOverOpen) {
     setSlideOverOpen(false)
     setShowArrows(false)
   }
+  if (
+    searchParams.get('v') &&
+    !slideOverOpen &&
+    data.length > 0 &&
+    selectedIndex !== undefined
+  ) {
```

Comment:
lemme check ... some of that logic guarded against returning to the project ops log after duplicating an ops log entry IIRC. That functionality is in [the "next" PR](https://github.com/AWeber-Imbi/imbi-ui/pull/118) but it could have slipped into this one when I split them.